rstudio/rstudio
# Update xterm.js to 4.8.1 to obtain fix for error logged in browser console

- Fixes #7645

### Intent

Scrolling terminal with mousewheel was writing a bunch of bogus errors to the browser console.

### Approach

This was an issue in xterm.js itself. Fixing by updating to the latest xterm.js.

### QA Notes

This is a very small xterm.js update (we were only two minor bugfix releases behind and in my experience xterm.js is very safe to update especially for these point releases. Xterm.js is the terminal emulator library we use (as does VSCode); it is only relevant to the terminal pane.

The fix here is purely to reduce noise in the browser console; no other known problem to test, so just quick sanity check of the terminal behavior is sufficient.
